src/core/src/OmniPolygonRasterOverlay.cpp

```diff
@@ -96,8 +102,9 @@ void OmniPolygonRasterOverlay::reload() {
         _pContext->getLogger()->error(error.message);
     };
 
+    auto debugInvert = getInvertSelection();
     _pPolygonRasterOverlay = new CesiumRasterOverlays::RasterizedPolygonsOverlay(
-        rasterOverlayName, polygons, invertSelection, *pEllipsoid, projection, options);
+        rasterOverlayName, polygons, debugInvert, *pEllipsoid, projection, options);
```

Comment:
Why is it called `debugInvert`?

Reply:
Thanks. I forgot to change that back. I was debugging inversion not working, fixed it, wrapped up for the day, and forgot this line.